content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+
+![ClusterManagement](/images/guides/kots/ha-cluster-cluster-mgmt.png)
+
+To get the commands to run on the other nodes, click on the 'Add Node' button. From here you can retrieve the command to use on Worker and Master nodes. For our exercise we will add Master nodes to give us a proper HA solution.
+
+![ClusterManagement](/images/guides/kots/ha-cluster-add-node-cmd.png)
+
+Copy the command and then ssh into nodes 02 & 03 and paste the command and hit enter. This should run a process similar to the initial install of the cluster and will take a few minutes. Once it is finished you should see something similar to the output below:
+
+![ClusterManagement](/images/guides/kots/ha-cluster-node-joined.png)
+
+### Verify Pods Are Running on All Nodes
+
+To verify that the nodes have in fact joined the cluster you can run the following commands:
+
+```shell
+kubectl get nodes
+```
+The output should show us all three nodes of our cluster and have a status of `READY`.
+
+```shell
+kubectl get pods --all-namespaces -o wide
+```
+
+The output should show that other than the default namespace, there are pods running on all three nodes.
+
+# Part II: Testing Scenarions & Troubleshooting
+
+There are various testing scenarios for an HA Cluster. For starters we are focusing on data retention/loss scenario. The sample application is already configured to write to the embedded Postgres StatefulSet.
+
+## Pre-Requisites
+
+This section assumes you followed the above steps in order and are using the sample application. Since we are testing data retention, make sure to use the routes described in the [Verify the Deployment](#verify-the-deployment) section.
+
+To verify that data is not lost, use pgAdmin as described above. Simply run the query again, and should be presented with the same results. You should also have a separate terminal in which to run:
+
+```shell
+watch -d 'kubectl get pods --all-namespaces -o wide'
+```
+
+The above command will display all of the pods in the cluster and on which node they are running on. By using `watch -d` the output will automatically get refreshed so you can watch pod activity as you delete pods and node.
+
+## Deleting a Pod
+
+If you followed along with the steps and sample application, you should see that the Postres pod is running on Node 01. In this test, we are going to delete the Postgres pod. The expected result is that a new pod will be scheduled (possibly on another node) and all the data in the database will be retained and not lost.
+
+To delete the Postgres pod deployed with the sample app, run the following command:
+
+```shell
+kubectl delete pod app-direct-postgresql-0
+```
+This command will schedule a new pod and then terminate the existing one. Verify that the new pod is up and running and check that the data is still there by using pgAdmin. You can also use the various routes in the sample app to verify database connectivity from the app.
```

Comment:
ditto re: psql

Reply:
Again, want to get this out the door, changed this whole section to be more generic. Happy to revisit this and add psql in the future.